build/ci/set-up-mms.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+# Copyright 2021 MongoDB Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+while getopts 'h:' opt; do
+  case ${opt} in
+  h) hostsFile="${OPTARG}" ;; # Output of Evergreen host.list
+  *) exit 1 ;;
+  esac
+done
+
+host=$(
+  cat <<EOF | python - "${hostsFile}"
+import sys
+import json
+with open(sys.argv[1]) as hostsfile:
+    hosts = json.load(hostsfile)
+    print(hosts[0]["dns_name"])
+EOF
+
+)
+
+cd ..
+cd ..
+
+
+echo "set service"
+  ./bin/mongocli config set service ops-manager
+
+echo "set ops_manager_url"
+  ./bin/mongocli config set ops_manager_url "http://${host}:9080/"
+
+echo "create first user"
+./bin/mongocli om owner create --firstName evergreen --lastName evergreen --email evergreenTest@gmail.com --password "evergreen1234_" -o json > apikeys.json
```

Comment:
[nit] couple of things here,
1) can we handle service and base url via ENV variables/evg expansions? we already do this for atlas and cloud manager

2) when creating the first user can we use a random password here? if not I'd rather handle that password via and evg expansion

3) instead of output as json you could use a go template in the toml format of the config and directly write the config from this command, check how we do something like this for gon, https://github.com/mongodb/mongocli/blob/79f568234a8243f4661294696d7a2ffd9930bf8b/build/ci/evergreen.yml#L213